modules/frontend/retry.go

```diff
@@ -51,6 +51,11 @@ func (r retryWare) Do(req *http.Request) (*http.Response, error) {
 			return resp, nil
 		}
 
-		span.LogFields(ot_log.String("msg", "error processing request"), ot_log.Int("try", triesLeft), ot_log.Error(err))
+		span.LogFields(
+			ot_log.String("msg", "error processing request"),
+			ot_log.Int("status_code", resp.StatusCode),
```

Comment:
CI caught this as well, but `resp` could be `nil` here

Reply:
Oh right, I wrote those tests myself 🤦

Reply:
Should be good now 👍🏻